src/DevBetterWeb.Web/Views/Shared/_Layout.cshtml

```diff
@@ -27,46 +27,39 @@
         <link rel="stylesheet" href="~/css/site.css" />
     </environment>
     <environment names="Staging,Production">
-        <link rel="stylesheet" href="https://ajax.aspnetcdn.com/ajax/bootstrap/3.3.7/css/bootstrap.min.css"
+        <link rel="stylesheet" href="https://ajax.aspnetcdn.com/ajax/bootstrap/4.3.1/css/bootstrap.min.css"
               asp-fallback-href="~/lib/bootstrap/dist/css/bootstrap.min.css"
               asp-fallback-test-class="sr-only" asp-fallback-test-property="position" asp-fallback-test-value="absolute" />
         <link rel="stylesheet" href="~/css/site.min.css" asp-append-version="true" />
     </environment>
     @await RenderSectionAsync("head", required: false).ConfigureAwait(false)
 </head>
 <body>
-    <div class="navbar navbar-default">
+    <nav class="navbar navbar-light bg-light static-top">
         <div class="container">
-            <div class="navbar-header">
-                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
-                    <span class="sr-only">Toggle navigation</span>
-                    <span class="icon-bar"></span>
-                    <span class="icon-bar"></span>
-                    <span class="icon-bar"></span>
-                </button>
-                <a asp-area="" asp-controller="Home" asp-action="Index" class="navbar-brand">
-                    <img src="~/images/logo_218x48.png" id="logo" alt="devBetter logo" />
-                    <img src="~/images/icon_50x50.png" id="icon" alt="devBetter icon" />
-                </a>
-            </div>
-            <div class="navbar-collapse collapse">
-                <ul class="nav navbar-nav">
-                    <li><a asp-area="" asp-controller="Home" asp-action="Index">Home</a></li>
-                    <li><a asp-area="" asp-page="/ArchivedVideos/Index">Videos</a></li>
-                    <li><a asp-area="" asp-page="/Questions">Questions</a></li>
-                </ul>
-            </div>
+            <a class="navbar-brand" href="/">
+                <img src="/images/logo_218x48.png" id="logo" alt="devBetter logo" />
+                <img src="/images/icon_50x50.png" id="icon" alt="devBetter icon" />
+            </a>
+            <a asp-page="/ArchivedVideos/Index">Videos</a>
+            <a asp-page="/Questions">Questions</a>
+            @if (User.IsInRole(Constants.Roles.ADMINISTRATORS))
```

Comment:
These View sections could be replaced with a Tag Helper like this one:
https://www.davepaquette.com/archive/2017/11/05/authorize-tag-helper.aspx

No need to do it for this PR but something to think about for another pass.